subsys/cpp/Kconfig

```diff
@@ -65,4 +66,6 @@ config RTTI
 	help
 	  This option enables support of C++ RTTI.
 
+endif # if NEWLIB_LIBC
```

Comment:
The organization here feels a bit weird. Maybe we could do this instead:

```
config LIB_CPLUSPLUS
	bool "Link with STD C++ library"
	depends on NEWLIB_LIBC
	help
	  Link with STD C++ Library.

if LIB_CPLUSPLUS

config EXCEPTIONS
	bool "Enable C++ exceptions support"
	help
	  This option enables support of C++ exceptions.

config RTTI
	bool "Enable C++ RTTI support"
	help
	  This option enables support of C++ RTTI.

endif # LIB_CPLUSPLUS
```

Might need to fix up some configuration files too, if they enable `EXCEPTIONS` without enabling `LIB_CPLUSPLUS`.

Reply:
Agreed, done.